src/offscreen/create-offscreen.ts

```diff
@@ -0,0 +1,12 @@
+import { config } from 'config'
+
+export const createOffscreen = async () => {
+  if (await chrome.offscreen.hasDocument()) {
+    await chrome.offscreen.closeDocument()
+  }
+  await chrome.offscreen.createDocument({
+    url: config.offscreen.url,
+    reasons: [chrome.offscreen.Reason.WEB_RTC],
```

Comment:
and websockets?

Reply:
Websockets are not listed in reason enum so I don't think we can/need to include it here